x/pubkey/types/params.go

```diff
@@ -0,0 +1,24 @@
+package types
+
+import (
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+const (
+	DefaultActivationLag = 25
+)
+
+// DefaultParams returns default pubkey module parameters.
+func DefaultParams() Params {
+	return Params{
+		ActivationLag: DefaultActivationLag,
```

Comment:
Maybe delay is more explicit. Wdyt?
I would suggest `activationBlockDelay` or `activationDelay` (but then in comment that it is in blocks and not time).

```suggestion
		ActivationBlockDelay: DefaultActivationBlockDelay,
```